src/beam/cli/login.py

```diff
@@ -48,8 +48,12 @@ def handle_login_request(token: str):
 
         text = "Do you want to create a new context? (y/n)"
         if terminal.prompt(text=text, default="n").lower() in ["y", "yes"]:
-            name = terminal.prompt(text="Enter context name", default=name)
+            context_name = terminal.prompt(text="Enter context name", default="")
```

Comment:
Don't provide a context name since we're prompting the user to create a new one. Also make sure the context name is not empty.